solidity/core/package.json

```diff
@@ -43,8 +43,9 @@
   "repository": "https://github.com/abacus-network/abacus-monorepo",
   "scripts": {
     "build": "hardhat compile && tsc",
+    "clean": "hardhat clean",
```

Comment:
Bonus points if you'd like to add a `yarn clean` job to the root while we're adding these. Or I can do it later :)